HUD/GetPauseMenuPosition.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: HUD
+aliases: ['0x5BFF36D6ED83E0AE', "_GET_PAUSE_MENU_CURSOR_POSITION"]
+---
+## GET_PAUSE_MENU_POSITION
+
+```c
+// 0x5BFF36D6ED83E0AE
+Vector3 GET_PAUSE_MENU_POSITION();
+```
+
+
+## Return value
+Z is always 0.
```

Comment:
Would it not be better to say in the return type that this is a Vec2?

Reply:
I put cs_type(Vector3) Vector2 in the new commit, that is fine right?

Reply:
What I meant was under `## Return value` say that this is used as a Vector2 so it can be implied that z is always 0